packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -458,6 +464,24 @@ class SwitchListTile extends StatelessWidget {
   /// If non-null, defines the background color when [SwitchListTile.selected] is true.
   final Color? selectedTileColor;
 
+  /// The horizontal gap between the titles and the leading/trailing widgets.
```

Comment:
Since this is just a duplicate of the corresponding property from ListTile, change ListTile's API doc by wrapping with 
```dart
/// {@template flutter.material.ListTile.horizontalTitleGap}
///  - doc goes here - 
/// {@endtemplate}

And then here you can include a copy of the same doc with
```dart
/// {@macro flutter.material.ListTile.horizontalTitleGap}
```

Likewise for minVerticalPadding and minLeadingWidth.